package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "graphql-kafka-subscriptions",
-  "version": "0.4.0",
+  "name": "@aerogear/graphql-kafka-subscriptions",
```

Comment:
We will need to release this as a new package..or should we do the fix upstream also (on https://github.com/ancashoria/graphql-kafka-subscriptions)

cc @wtrocki @machi1990

Reply:
We can release as new package for testing and then suggest the changes upstream, but having a new package in the meantime may unblock some things on our end that depend on this fix.